dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.42/V2_42_34__Create_sequence_for_note_table.sql

```diff
@@ -0,0 +1,5 @@
+create sequence if not exists note_id_sequence;
```

Comment:
we usually name sequences as "tablename_sequence"
So maybe this can be "note_sequence" ?

Reply:
Done